.gitignore

```diff
@@ -0,0 +1,9 @@
+# Include your project-specific ignores in this file
+# Read about how to use .gitignore: https://help.github.com/articles/ignoring-files
+
+build
+node_modules
+ncp-debug.log
+npm-debug.log
+restructure/dist
+.DS_Store
```

Comment:
This isn't really a project-specific file. We should all have this in a global .gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore